public/components/overview/home.tsx

```diff
@@ -3,159 +3,272 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import React, { useState, useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { HashRouter, RouteComponentProps, Switch, Route } from 'react-router-dom';
-import { EuiSpacer, EuiText, EuiFlexGroup, EuiFlexItem, EuiCard } from '@elastic/eui';
-import { TraceAnalyticsCoreDeps } from '../trace_analytics/home';
+import { EuiText } from '@elastic/eui';
+import moment from 'moment';
 import { ChromeBreadcrumb } from '../../../../../src/core/public';
-import { coreRefs } from '../../../public/framework/core_refs';
+import { coreRefs } from '../../framework/core_refs';
+import { ContentManagementPluginStart } from '../../../../../src/plugins/content_management/public';
+import { HOME_CONTENT_AREAS, HOME_PAGE_ID } from '../../plugin_helpers/plugin_overview';
+import { cardConfigs, GettingStartedConfig } from './components/card_configs';
+import { uiSettingsService } from '../../../common/utils';
+import { AddDashboardCallout } from './components/add_dashboard_callout';
+import { DashboardControls } from './components/dashboard_controls';
+import { SelectDashboardFlyout } from './components/select_dashboard_flyout';
+import { setNavBreadCrumbs } from '../../../common/utils/set_nav_bread_crumbs';
+import './index.scss';
 
+// Plugin IDs
 const alertsPluginID = 'alerting';
 const anomalyPluginID = 'anomalyDetection';
 
-// Plugin URLs
-const gettingStartedURL = 'observability-gettingStarted';
-const discoverURL = 'data-explorer';
-const metricsURL = 'observability-metrics';
-const tracesURL = 'observability-traces-nav#/traces';
-const alertsURL = 'alerting';
-const anomalyDetectionURL = 'anomaly-detection-dashboards';
-
-const checkIfPluginsAreInstalled = async (
-  setAlertsPluginExists: React.Dispatch<React.SetStateAction<boolean>>,
-  setAnomalyPluginExists: React.Dispatch<React.SetStateAction<boolean>>
-) => {
-  try {
-    const response = await fetch('../api/status', {
-      headers: {
-        'Content-Type': 'application/json',
-        'osd-xsrf': 'true',
-        'accept-language': 'en-US,en;q=0.9,zh-CN;q=0.8,zh;q=0.7,zh-TW;q=0.6',
-        pragma: 'no-cache',
-        'sec-fetch-dest': 'empty',
-        'sec-fetch-mode': 'cors',
-        'sec-fetch-site': 'same-origin',
-      },
-      method: 'GET',
-      referrerPolicy: 'strict-origin-when-cross-origin',
-      mode: 'cors',
-      credentials: 'include',
-    });
-    const data = await response.json();
-
-    let alertsExists = false;
-    let anomalyExists = false;
+const uiSettingsKey = 'observability:defaultDashboard';
```

Comment:
We should move these to `common/constants/overview`  folder

Reply:
1